### DISCUSSION TITLE
Resource Usage for multiple sites on same server (migrating from ee 3)

### DISCUSSION THREAD

**Discussion by ssuess** [EasyEngine/easyengine#1604] [Support Requests] [answered]
Hi, I am about to migrate some older ee 3 sites, and I was trying to decide whether to go with Wordops (https://wordops.net/) or ee 4, and I have the following questions:
1. ee4 seems easy to setup, but if I have several websites (6 low traffic sites) on the same server, does it use a lot more resources that ee 3? (because of all the docker containers setup for each site?)
2. Would it be better to go with Wordops (a fork of ee3)? Does it use less resources per machine?
3. Any other advice before migrating?

Thanks!

**Comment by dhsathiya** [comment, marked as answer]
Hi @ssuess,

Here are the answers based on my experience on EE4 (I am EE4 maintainer)

1. Yes EE4 requires slightly higher resources as it is using containers. But it is not that high. You won't need an extra GB of RAM for sure.
2. WordOps is also a great tool. The major benefit of containers is that in case one site gets infected, at least other sites will be safe. Instead of hunting down the infected files you can just disable and enable site to get a new copy of whole FS.
3. I suggest buying 2 smaller VPS and use EE4 and WO on each and then migrate one site. Keep them live for a month and decide what's best for you :)

Talking about the numbers - I am managing 20 low traffic / idle sites on 1vCPU - 2 GB RAM. So I don't think there is much needed to check in terms of resources required. I suggest adding an extra CPU and a GB of RAM. Rest depends on traffic, DB size and Code.

**Reply by ssuess** [reply]
Thanks this is VERY helpful!
